src/main.rs

```diff
@@ -78,36 +79,54 @@ async fn main() -> Result<()> {
                     None => PullDestination::MainStore,
                 };
                 let (sources, docker_config) = remote_server_options(matches)?;
-                pull::pull(uri, docker_config, sources, destination).await?;
+                let (key_files, annotations) = verification_options(matches)?;
+
+                // verify policy prior to pulling if keys listed, and keep the
+                // verified manifest digest of last iteration, even if all are
+                // the same:
+                let mut verified_manifest_digest: String = String::new();
+                for key_file in key_files.iter() {
+                    // let key = verify::read_key_file(&key_file[0])?;
+                    verified_manifest_digest = verify::verify(
+                        uri,
+                        docker_config.clone(),
+                        sources.clone(),
+                        annotations.clone(),
+                        &key_file[0],
+                    )
+                    .await?;
```

Comment:
Now that `verified_manifest_digest` is an `Option`, you just have to change this line to `await?.ok();`

Reply:
Fixed, thanks!